### ISSUE TITLE
Feature Request player areas

### ISSUE THREAD

**Issue by johnwoltman** [overviewer/Minecraft-Overviewer#1567]
Would it be possible to have Overviewer render only a certain radius around each player?  That would keep most of the world mysterious, but still allow people to, for example, see their houses or their latest adventure.  Maybe it's already possible through some custom configuration, but I couldn't figure it out from the docs.

**Comment by tswsl1989** [comment]
I don't believe this is possible currently, as we handle players and other points of interest in a seperate process to the main map render.
The closest you can manage (as far as I can see) would be to define renders for specific sets of coordinates using the crop parameter - see http://docs.overviewer.org/en/stable/config/#rendering and the warnings contained there.

**Comment by ilar** [comment]
I actually had a thought about how this could be accomplished. I was trying to think of a way to pre-generate the world using worldborder, but not show them on the map. The solution there is to provide a chunk format Lastupdate for which chunks less that were updated before that than do not render. However, a crude version of the above could be implemented by having the provided Lastupdate be dynamic- e.g. so not render any chunks that were not updated in the last x minutes of ticks. A less crude version would cache and compare inhabited times to see if the chunk has been inhabited and not just modified/saved.
